src/index.html

```diff
@@ -1,14 +1,75 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo__link">
+        <a class="logo__img" href="#">
+          <img
+            src="./images/logo.png"
+            alt="Moyo logo"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="list__items">
+          <li class="list__item">
+            <a
+              href="#"
+              class="item__link
+              is-active"
+            >Apple</a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="item__link">Samsung</a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="item__link">Smartphones</a>
+          </li>
+
+          <li class="list__item">
+            <a
+              href="#"
+              class="item__link"
+              data-qa="hover"
+            >Laptops & Computers
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="item__link">Gadgets</a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="item__link">Tablets</a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="item__link">Photo</a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="item__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
   </body>
+
 </html>
```

Comment:
Don't add blank lines between parent and child elements. Only between siblings
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/1dc494dc-9fd2-477e-ad71-c1d6a78d5abb)